transformations/concept2sentence/transformation.py

```diff
@@ -0,0 +1,112 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+from sibyl import Concept2Sentence
+
+
+class C2S(SentenceAndTargetOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION
+    ]
+    languages = ["en"]
+    keywords = [
+        # type of linguistic change
+        "lexical", 
+        "semantic",
+        # type of algorithm
+        "model-based", 
+        "transformer-based", 
+        "api-based", 
+        "tokenizer-required",
+        # naturalness of the generation
+        "unnaturally-written",
+        # potential accuracy & precision of the generation
+        "possible-meaning-alteration",
+        "low-precision",
+        "high-coverage",
+        "high-generations"
+    ]
+
+    def __init__(self, 
+                 seed=0, 
+                 max_outputs=1, 
+                 dataset=None, 
```

Comment:
Okay, this looks good to me overall - can you just add a default dataset here? And seems you were unable to get the tests working because of that. You can take a look [here](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/filters/length/test.json) how args can be specified in your test.json.

Reply:
I don't think it would be wise to have a default dataset. The dataset argument is used to download a fine-tuned transformer from huggingface_hub, which will in turn be used to identify the most salient tokens in the input for concept selection. However, that's only relevant for that particular dataset. When dataset is None, the transformation uses a lightweight graph-based approach to pick interesting concepts from the input - its more general and less likely to cause issues for users down the road. 

I removed the tests because the transformation picks a random number of concepts to use each time to boost diversity, so the output is potentially different each run.